manifests/deis-logger-fluentd-daemon.yaml

```diff
@@ -33,10 +33,6 @@ spec:
         env:
         - name: DEBUG
           value: "true"
-        - name: SYSLOG_HOST
```

Comment:
That's what I assumed. But if so, then this stanza should also be removed from the manifest in this repo, since it's not in workflow-dev:
```yaml
        securityContext:
          privileged: true
```